deploy/build-env-public.sh

```diff
@@ -0,0 +1,9 @@
+WORKDIR=${1:-"/opt/apps/track-web-public"}
+mkdir -p $WORKDIR
+cd $WORKDIR
+python3 -m venv .venv
```

Comment:
maybe to make sure there is not conflict , have a different virtual env name for the public package.